dist/formats/financial_releases_campaign/publisher_v2/schema.json

```diff
@@ -258,6 +258,27 @@
           }
         }
       }
+    },
+    "asset_link": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "url",
+        "content_type",
+        "alt_text"
+      ],
+      "properties": {
+        "url": {
+          "type": "string",
+          "format": "uri"
+        },
+        "content_type": {
```

Comment:
@boffbowsh This format will have a pdf document uploaded to it too. Very soon. Does that mean we will need to have content type then ?  
If not, I can remove it. I am not really using the content type in the app.